### ISSUE TITLE
The setup.py in windows can cause a encoding error.

### ISSUE THREAD

**Issue by a358003542** [springload/draftjs_exporter#98]
This will be corrected by : 

```
with open('README.md', encoding='utf8') as f:
    long_description = f.read()
```

**Comment by thibaudcolas** [comment]
Hey @a358003542, thanks for raising this.

I'm not sure how best to test this but I'll try my best to make a fix.

**Comment by thibaudcolas** [comment]
@a358003542 this should now be released as v2.1.2 on PyPI if you want to give it another try.
